drivers/ieee802154/ieee802154_rf2xx.h

```diff
@@ -89,6 +89,7 @@ struct rf2xx_dt_spi_t {
 
 struct rf2xx_config {
 	u8_t inst;
+	u8_t has_mac;
```

Comment:
could you move these 2 attributes at the end of the structure, for better (future) alignment ?

Reply:
done!